HISTORY.md

```diff
@@ -1,5 +1,10 @@
 ## History
 
+### 1.5.4
+
+* Combine getXSTSToken and getXSTSTokenWithTitle
```

Comment:
Please use the name of the PR along with PR number in changelog, otherwise these changes sound like a documented API change.

```
Combine `getXSTSToken` & `getXSTSTokenWithTitle` and
 implement own `getUserToken`  (#47)
```

Reply:
Donezo :)